tests/conftest.py

```diff
@@ -70,3 +73,361 @@ def _sample_all_wsis(_sample_ndpi, _sample_svs, _sample_jp2, tmpdir_factory):
+        .join("sample_patch_extraction.csv")
+
+    if not pathlib.Path(csv_file_path).is_file():
+        print("\nDownloading sample patch extraction csv file")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata/"
+                         "patchextraction/sample_patch_extraction.csv")
+        with open(csv_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping sample patch extraction csv file")
+
+    return csv_file_path
+
+
+@pytest.fixture(scope="session")
+def _patch_extr_json(tmpdir_factory):
+    """Sample pytest fixture for sample patch extraction json.
+    Download sample patch extraction json for pytest."""
+
+    json_file_path = tmpdir_factory.mktemp("data")\
+        .join("sample_patch_extraction.json")
+
+    if not pathlib.Path(json_file_path).is_file():
+        print("\nDownloading sample patch extraction json file")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata/"
+                         "patchextraction/sample_patch_extraction.json")
+        with open(json_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping sample patch extraction json file")
+
+    return json_file_path
+
+
+@pytest.fixture(scope="session")
+def _patch_extr_npy(tmpdir_factory):
+    """Sample pytest fixture for sample patch extraction npy.
+    Download sample patch extraction npy for pytest."""
+
+    npy_file_path = tmpdir_factory.mktemp("data")\
+        .join("sample_patch_extraction.npy")
+
+    if not pathlib.Path(npy_file_path).is_file():
+        print("\nDownloading sample patch extraction npy file")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata/"
+                         "patchextraction/sample_patch_extraction.npy")
+        with open(npy_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping sample patch extraction npy file")
+
+    return npy_file_path
+
+
+@pytest.fixture(scope="session")
+def _patch_extr_noheader(tmpdir_factory):
+    """Sample pytest fixture for sample patch extraction noheader csv.
+    Download sample patch extraction noheader csv for pytest. """
+
+    noheader_file_path = tmpdir_factory.mktemp("data")\
+        .join("sample_patch_extraction-noheader.csv")
+
+    if not pathlib.Path(noheader_file_path).is_file():
+        print("\nDownloading sample patch extraction no header csv file")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata/"
+                         "patchextraction/sample_patch_extraction-noheader.csv")
+        with open(noheader_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping sample patch extraction no header csv file")
+
+    return noheader_file_path
+
+
+@pytest.fixture(scope="session")
+def _patch_extr_2col_json(tmpdir_factory):
+    """Sample pytest fixture for sample patch extraction 2col json.
```

Comment:
"""Sample pytest fixture for sample patch extraction 2col json.
    Download sample patch extraction 2col json for pytest.

"""

Reply:
Resolved in this commit https://github.com/TIA-Lab/tiatoolbox/pull/99/commits/f140ef4378c6618ae9d1f5cc7e47785c9c6da702